docs/Write-Plugin.md

```diff
@@ -0,0 +1,58 @@
+<h1 align="center">Fastify</h1>
+
+# How to write a good plugin
+First of all thank you for having decided to build a plugin for Fastify. Fastify is a *battery not included* framework and plugins are its strength, so thank you.<br>
+As you probably know, Fastify has been built since the beginning with performances and low overhead in mind, so if you are building a plugin you should keep a special eye on those. In this document we will analyze what characterizes a good quality plugin.
+
+*Need some inspiration? You can use the label ["plugin suggestion"](https://github.com/fastify/fastify/issues?q=is%3Aissue+is%3Aopen+label%3A%22plugin+suggestion%22) in our issue tracker!*
+
+## Code
+Fastify uses different techniques to optimize its code, many of them are documented in our Guides. We highly recommend you to read [the hitchhiker's guide to plugins](https://github.com/fastify/fastify/blob/master/docs/Plugins-Guide.md) to discover all the APIs you can use to build your plugin and learn how use them to the best possible.
```

Comment:
Hmm, "Guides" should really be a link, but do we have an index of "Guides"?

Strike "to the best possible".

Reply:
I'd say "getting started" e "plugins guide".